### DISCUSSION TITLE
Welcome to technical_art Discussions!

### DISCUSSION THREAD

**Discussion by functionguyy** [functionguyy/technical_art#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Isoghena** [comment]
The article on C_variables is very detailed and simplified. However, i would like to know the if there is a difference between data types and types.

**Reply by functionguyy** [reply]
Hi, @Isoghena thanks for your comment. However, there is a slight difference between data types and types. Type refers to a specification. Data type refers to data that fits a type specification.

So when we declare an `int` **type** variable, we're instructing the compiler to make a variable that can store data types that fit the `int` type specification.

For more info read, [here](https://cs.stackexchange.com/a/119296)
